cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/ThroughputLatencyGraphs/parser.ts

```diff
@@ -44,10 +44,10 @@ interface IRawMetricData {
 }
 
 const METRIC_NAME_MAP = {
-  'user.dml.insert': 'inserts',
-  'user.dml.update': 'updates',
-  'user.dml.delete': 'deletes',
-  'user.dml.error': 'errors',
+  'user.dml.inserts': 'inserts',
```

Comment:
nit: Seems like all the parsers have most of the metrics in common. Can we extract out the common metrics into one parser and reuse it. Would be useful if at some point in we change any of the metrics.

Reply:
the parsers are different, the grouping of the metrics are different. There is some clean up and consolidation that can happen, will do that in subsequent PR